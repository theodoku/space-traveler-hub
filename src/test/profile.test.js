import ProfileMission from '../components/ProfileMisison';

describe('Testing the profile', () => {
  const component = (
    <ProfileMission mission={{
      mission_id: '1',
      mission_name: 'Test ',
      description: 'Test  description',
      reserved: true,
    }}
    />
  );

  test('should match snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
