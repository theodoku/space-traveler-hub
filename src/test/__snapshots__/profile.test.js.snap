// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the profile should match snapshot 1`] = `
<ProfileMission
  mission={
    Object {
      "description": "Test  description",
      "mission_id": "1",
      "mission_name": "Test ",
      "reserved": true,
    }
  }
/>
`;
